documentation/modules/ROOT/pages/development/engine.adoc

```diff
@@ -20,6 +20,17 @@ They still want the same data change events, but prefer to have the connectors s
 
 This `debezium-api` module defines a small API that allows an application to easily configure and run {prodname} connectors using {prodname} Engine.
 
+Beginning with the 2.6.0 release, {prodname} provides two implementations of the `DebeziumEngine` interface.
+The older one, `EmbeddedEngine`, processes all records sequentially and allows to run on one task for given connector.
+This the default implementation.
+Beginning with the 2.6.0 release, a new `AsyncEmbeddedEngine` implementation is available.
+This implementation processes records in multiple threads, and can run multiple tasks, if the connector supports it (currently only the connectors for SQL Server and MongoDB support running multiple tasks within a single connector).
```

Comment:
```suggestion
This implementation also runs only a single connector, but it can process records in multiple threads, and run multiple tasks, if the connector supports it (currently only the connectors for SQL Server and MongoDB support running multiple tasks within a single connector).
```